frameworks/elastic/universe/config.json

```diff
@@ -592,6 +592,22 @@
         "count"
       ]
     },
+    "sgadmin_nodes": {
+      "description": "Elasticsearch sgadmin node configuration properties",
+      "type": "object",
+      "properties": {
+        "count": {
+          "description": "Number of sgadmin nodes to run",
+          "type": "integer",
+          "default": 0,
+          "minimum": 0,
+          "maximum": 1
```

Comment:
If `1` is the max, then do we ever need user input to specify node count? If they set `searchguard_enabled` to true, can't we just deploy 1 node for them knowing that's all they can ever have?

Reply:
yes, it seems this is the right thing to do, thanks